docs/doxygen/CMakeLists.txt

```diff
@@ -41,4 +41,4 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #######################################################################
 
-add_doxygen_target(chai_doxgen)
+blt_add_doxygen_target(chai_doxgen)
```

Comment:
Is this because of a BLT update?

Reply:
Correct. I updated blt to the same commit as Umpire.